tests/components/azure_devops/test_config_flow.py

```diff
@@ -200,16 +192,16 @@ async def test_reauth_project_error(hass: HomeAssistant) -> None:
         assert result2["errors"] == {"base": "project_error"}
 
 
-async def test_reauth_flow(hass: HomeAssistant) -> None:
+async def test_reauth_flow(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+) -> None:
     """Test reauth works."""
     with patch(
         "homeassistant.components.azure_devops.config_flow.DevOpsClient.authorize",
         return_value=False,
     ):
```

Comment:
This patch is not needed anymore if you include `mock_devops_client` in the parameters of this test

Reply:
I think you have a big code cleanup if you apply this to all tests. You can mutate the mock in the test itself via

```py
mock_devops_client.authorized = False
```

For example